<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frame_servo_controll</class>
 <widget class="QGroupBox" name="frame_servo_controll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>73</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="title">
   <string>ID:1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Угол:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QSlider" name="horizontalSlider_value">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;значение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_value">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;значение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>500</number>
        </property>
        <property name="maximum">
         <number>2500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="spinBox_max_value">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;максимум&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
     <property name="value">
      <number>2500</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBox_min_value">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;минимум&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Серва:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_min_angle">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;манимальный угол&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_max_angle">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;максимальный угол&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QSlider" name="horizontalSlider_angle">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;значение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_angle">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;максимальный угол&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>-180.000000000000000</double>
        </property>
        <property name="value">
         <double>-180.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_remove">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>spinBox_min_value</tabstop>
  <tabstop>horizontalSlider_value</tabstop>
  <tabstop>spinBox_value</tabstop>
  <tabstop>spinBox_max_value</tabstop>
  <tabstop>doubleSpinBox_min_angle</tabstop>
  <tabstop>horizontalSlider_angle</tabstop>
  <tabstop>doubleSpinBox_angle</tabstop>
  <tabstop>doubleSpinBox_max_angle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
